app/components/ScenariosManager/IsolationTester/IsolationTester.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect } from "react";
+import { Flex, Button, message, InputNumber, Collapse } from "antd";
+import { Scenario } from "@/app/types/scenario";
+import { getCSVTests } from "@/app/utils/api";
+import { RuleMap } from "@/app/types/rulemap";
+import ScenarioFormatter from "../ScenarioFormatter";
+import styles from "./IsolationTester.module.css";
+import { DecisionGraphType } from "@gorules/jdm-editor";
+import { valueType } from "antd/es/statistic/utils";
+
+interface IsolationTesterProps {
+  scenarios: Scenario[];
+  resultsOfSimulation: Record<string, any> | null | undefined;
+  simulationContext?: Record<string, any>;
+  setSimulationContext: (data: any) => void;
+  resetTrigger: boolean;
+  jsonFile: string;
+  rulemap: RuleMap;
+  scenarioName?: string;
+  ruleContent?: DecisionGraphType;
+  ruleVersion?: string | boolean;
+}
+
+export default function IsolationTester({
+  scenarios,
+  resultsOfSimulation,
+  simulationContext,
+  setSimulationContext,
+  resetTrigger,
+  jsonFile,
+  rulemap,
+  scenarioName,
+  ruleContent,
+  ruleVersion,
+}: IsolationTesterProps) {
+  const [simulationRun, setSimulationRun] = useState(false);
+  const [scenarioExpectedOutput, setScenarioExpectedOutput] = useState({});
+  const [editingScenario, setEditingScenario] = useState(scenarioName && scenarioName.length > 0 ? true : false);
```

Comment:
These aren't really used? I think you can remove them and adjust/remove the useffects below.

Reply:
Missed when copying over from the Scenarios Generator and Viewer components! Thank you, definitely unused.